### PR TITLE
Updates to setup process

### PR DESCRIPTION
**Pull request by arwagner**
- Sample token files
- Update README
- Get rid of references to files that are not needed
- etc.

### CODE REVIEW

File: config/testconfig.js [hunk 1]
```diff
@@ -18,12 +17,10 @@ config["welcome"]["channel"] = "unstable_bot";
 config["leagues"]["45+45"]["spreadsheet"]["key"] = "1BeRN76zaB_uCCrCra2yTEEw_r6C5_b8P59aN_BrJsyA";
 config["leagues"]["45+45"]["spreadsheet"]["serviceAccountAuth"] = {
```
Comment (lakinwecker) [inline config/testconfig.js]: Can we completely get rid of serviceAccountAuth? I don't think we use the spreadsheets at all anymore, right?

File: config/config.js [hunk 2]
```diff
@@ -55,8 +54,7 @@ var config = {
             "spreadsheet": {
                 "key": "1cTBOQNrbUw9-RWmrC9b6Yyi3i2MtIvUWuXNxzQmGfSs",
                 "serviceAccountAuth": {
```
Comment (lakinwecker) [inline config/config.js]: Sam with here.

File: config/config.js [hunk 3]
```diff
@@ -55,8 +54,7 @@ var config = {
             "spreadsheet": {
```
Comment (endrawes0) [inline config/config.js]: I wont ask you to do this, but we should be able to drop the entire spreadsheet section of the config - as soon as we remove the dependencies on src/spreadsheets.js

File: config/testconfig.js (hunk 1 above)
Comment (endrawes0) [inline config/testconfig.js]: confirmed - its unneeded 👍

File: config/config.js (hunk 3 above)
Comment (endrawes0) [inline config/config.js]: confirmed - its unneeded

**Comment by endrawes0** [comment]
Thanks for the contribution - will help significantly with onboarding new devs.

**Pull request merged**